honeybee_3dm/tojson.py

```diff
@@ -0,0 +1,39 @@
+try:  # import the core honeybee dependencies
+    from honeybee.model import Model
+    from honeybee.config import folders
+except ImportError as e:
+    raise ImportError('\nFailed to import honeybee:\n\t{}'.format(e))
+
+import os
+import json
+
+
+def publish_json(_hb_objs, _name_=None, _folder_=None, indent_=None, abridged_=False, _dump=True):
```

Comment:
This is how we name arguments inside the Grasshopper components to clarify which ones are required. In the python code we don't use `_` before and after the argument names.

Reply:
Complied.